pkg/diffs/diffs.go

```diff
@@ -229,3 +230,26 @@ func GetPresubmitsForClusterProfiles(prowConfig *prowconfig.Config, profiles []c
 	}
 	return ret
 }
+
+// GetChangedPeriodics returns the changed presubmits mapped by their name.
+func GetChangedPeriodics(prowMasterConfig, prowPRConfig *prowconfig.Config, logger *logrus.Entry) []prowconfig.Periodic {
+	masterPeriodics := getPeriodicsPerName(prowMasterConfig.JobConfig.AllPeriodics())
+	prPeriodics := getPeriodicsPerName(prowPRConfig.JobConfig.AllPeriodics())
+
+	var changedPeriodics []prowconfig.Periodic
+	for name, job := range prPeriodics {
+		if !reflect.DeepEqual(masterPeriodics[name], job) {
```

Comment:
@stevekuznetsov 
I forced to use this instead of equality's DeepEqual because of, guess what.... It panics because it can't process the `time.Duration` in decoration config :)... I tried also to use [reflect#Equalities.DeepDerivative](https://godoc.org/k8s.io/apimachinery/third_party/forked/golang/reflect#Equalities.DeepDerivative) instead but still panics... Any ideas?